### ISSUE TITLE
[mu4e] *mu4e-loading* window doesn't disappear

### ISSUE THREAD

**Issue by korayal** [syl20bnr/spacemacs#13314]
#### Description :octocat:
`*mu4e-loading*` doesn't disappear when I view an email

PS. I'm using the latest master branch of `mu4e` as of date.

#### Reproduction guide :beetle:
- Start Emacs
- press `SPC a M` keys and select an email message

*Observed behaviour:* :eyes: :broken_heart:

Two windows open up (`*mu4e-view*`, and `*mu4e-loading*`), the latter takes over
most of the view space and never disappears.

*Expected behaviour:* :heart: :smile:

I would expect the *mu4e-loading* window(& buffer) to disappear

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. a4459dd28)
- Graphic display: t
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
(typescript systemd
            (rust :variables rust-backend 'racer rust-format-on-save t)
            shell-scripts csv helm
            (auto-completion :variables auto-completion-enable-sort-by-usage t auto-completion-enable-help-tooltip t auto-completion-enable-snippets-in-popup t auto-completion-private-snippets-directory "~/.spacemacs.d/snippets")
            semantic themes-megapack ibuffer ranger fasd
            (mu4e :variables mu4e-installation-path "/usr/local/share/emacs/site-lisp/mu4e" mu4e-enable-async-operations t mu4e-enable-mode-line t)
            restclient emacs-lisp git
            (markdown :variables markdown-live-preview-engine 'vmd)
            (multiple-cursors :variables multiple-cursors-backend 'evil-mc)
            github html
            (c-c++ :variables ycmd-python-binary-path "/home/korayal/.virtualenvs/soostone3.7/bin/python" ycmd-server-command
                   '("python3.7" "-u" "/home/korayal/test/EMACS/ycmd/ycmd"))
            (javascript :variables js2-basic-offset 2 js-indent-level 2 node-add-modules-path t)
            coffeescript purescript
            (typescript :variables typescript-indent-level 2 typescript-backend 'tide tide-tsserver-executable "/home/korayal/.nvm/versions/node/v12.0.0/bin/tsserver" typescript-linter 'tslint typescript-fmt-tool 'prettier)
            neotree
            (org :variables org-enable-org-journal-support t org-enable-github-support t org-projectile-file "TODOs.org" org-enable-sticky-header t)
            (lsp :variables lsp-python-use-init-for-project-root t)
            (python :variables python-backend 'anaconda python-auto-set-local-pyvenv-virtualenv 'on-visit python-test-runner
                    '(nose pytest))
            ipython-notebook
            (haskell :variables haskell-enable-hindent t haskell-completion-backend 'dante haskell-stylish-on-save nil haskell-process-type 'cabal-new-repl haskell-stack-ghc "runghc")
            yaml
            (shell :variables shell-default-height 30 shell-default-position 'bottom shell-default-full-span nil shell-default-shell 'term shell-default-term-shell "/usr/bin/fish")
            syntax-checking
            (version-control :variables version-control-diff-tool 'git-gutter+ version-control-diff-side 'left version-control-global-margin t)
            pdf epub better-defaults colors
            (evil-snipe :variables)
            docker nixos dhall)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS GLIB NOTIFY LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS LIBSYSTEMD LCMS2

**Comment by korayal** [comment]
Looks like this happens only when I enable `dotspacemacs-switch-to-buffer-prefers-purpose`

**Comment by duianto** [comment]
You can close this issue if the following PR fixed it:
[#1598] [mu4e] Fix purpose labels for mu4e buffers #13331
